.github/labeler.yaml

```diff
@@ -0,0 +1,20 @@
+dashboard:
+  - dashboard/**/*
+
+documentation:
+  - any:
+      - docs/**/*
+      - '!docs/docs/reference/docgen/**/*'
+  - examples/**/*
```

Comment:
Should we have examples as a separate label?

Reply:
yes, done